File: api/src/main/java/com/mystore/manager/api/common/payload/GlobalDataPayload.java
package com.mystore.manager.api.common.payload;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class GlobalDataPayload extends GlobalUserDatePayload {
    private String code;
    private String name;

    public String getFullName() {
        return code + " - " + name;
    }
}

File: ui-admin/src/app/backend/payloads/business/global/GlobalDataPayload.ts
import { GlobalUserDatePayload } from "./GlobalUserDatePayload";

export class GlobalDataPayload extends GlobalUserDatePayload{
    code: string;
    name: string;
    
    constructor(code: string | null = null, name: string | '' = '' ){
        super();
        this.code = code;
        this.name = name;
    }
}